content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -46,7 +46,8 @@ Each request can be recorded with an associated _stage_. The defined stages are:
 - `Panic` - Events generated when a panic occurred.
 
 {{< note >}}
-Audit events are different from the
+An audit [Event](/docs/reference/config-api/apiserver-audit.v1/#audit-k8s-io-v1-Event)
```

Comment:
nit: You may want to update this link text to something more meaningful.
Something like:  "Audit Event configuration"?

Reply:
Done.